File: functionapi-tools/src/main/java/io/github/cottonmc/functionapi/tools/commands/EntityAttributeCommand.java
package io.github.cottonmc.functionapi.tools.commands;

import com.mojang.brigadier.CommandDispatcher;
import com.mojang.brigadier.builder.LiteralArgumentBuilder;

import java.util.Map;

public class EntityAttributeCommand {

    public static void register(CommandDispatcher<Map<String, Object>> commandDispatcher) {

        LiteralArgumentBuilder<Map<String, Object>> addentityAttribute = LiteralArgumentBuilder.literal("entityattribute");



    }

}

File: functionapi-items/src/main/java/io/github/cottonmc/functionapi/content/items/commands/EntityAttributeCommand.kt
package io.github.cottonmc.functionapi.content.items.commands

import com.mojang.brigadier.CommandDispatcher
import com.mojang.brigadier.builder.LiteralArgumentBuilder
import io.github.cottonmc.functionapi.api.content.CommandRegistrator

class EntityAttributeCommand : CommandRegistrator {
    override fun register(commandDispatcher: CommandDispatcher<Map<String, Any>>) {
        val addentityAttribute = LiteralArgumentBuilder.literal<Map<String, Any>>("entityattribute")
    }
}